### ISSUE TITLE
Question: How hard would it be to ignore identical changes?

### ISSUE THREAD

**Issue by acote-coveo** [softwaremill/diffx#203]
I have a use case where I would like to only log the changes between my objects over time.

I would like to know if it's easy to simply ignore identical results?

**Comment by ghostbuster91** [comment]
Hi, it should be fairly easy. I will try to take a look at it during the upcoming days.

**Comment by nicolasdalsass** [comment]
I used something like this client side
```
val compareResult = compare(object1, object2).asInstanceOf[DiffResultObject]
val onlyDiffs =  compareResult
        .copy(fields = compareResult.fields.filter(!_._2.isInstanceOf[Identical[_]]))
```
Works pretty well, for the use case where you're certain the top level compare returns an object comparison

**Comment by ghostbuster91** [comment]
Thanks @nicolasdalsass  that really seems like a simple and good enough approach. I added it to the library and also added support for sets and maps.

**Comment by ghostbuster91** [comment]
Released as 0.4.2. I will close that issue but feel free to open a new one if this solution doesn't fit your needs.
